src/Umbraco.Web.Common/Security/MemberManager.cs

```diff
@@ -28,5 +28,8 @@ public class MemberManager : UmbracoUserManager<MemberIdentityUser, MemberPasswo
                 services, logger, passwordConfiguration)
         {
         }
+
+        public bool IsMemberAuthorized(IEnumerable<string> allowTypes = null, IEnumerable<string> allowGroups = null, IEnumerable<int> allowMembers = null)
+            => true; // TODO: Implement!
```

Comment:
What is stopping us from implementing this now?

Reply:
Doing this as part of public access